core/src/main/java/io/undertow/server/handlers/proxy/mod_cluster/MCMPConfig.java

```diff
@@ -164,20 +145,16 @@ public int getAdvertiseFrequency() {
             return advertiseFrequency;
         }
 
-        public String getManagementHost() {
-            return managementHost;
-        }
-
-        public int getManagementPort() {
-            return managementPort;
+        public InetSocketAddress getManagementSocketAddress() {
```

Comment:
Ideally yes, buts its really bad :-( and I am not sure how else to do this cleanly.. also all the getters are used internally by the implementation so there should be no harm.